socorro/unittest/external/postgresql/unittestbase.py

```diff
@@ -19,11 +21,15 @@ class PostgreSQLTestCase(TestCase):
     required_config = Namespace()
     required_config.namespace('database')
     required_config.add_option(
+        name='database_url',
+        default='postgres://test:aPassword@localhost:5432/socorro_integration_test',
+        doc='Name of database to manage',
+    )
+    required_config.add_option(
```

Comment:
Exploration revealed depedencies on `socorro.database.database` in our tests and middleware. Going back to just supporting the minimal change of adding support for `database_url` without deprecating DSN, pending deprecation of `socorro.database.database`.